tests/flow/test_config.py

```diff
@@ -131,3 +131,97 @@ def test08_config_restore_timeout_to_default(self):
         # Issue long-running query to validate the reconfiguration
         result = redis_graph.query("UNWIND range(1,1000000) AS v RETURN COUNT(v)")
         self.env.assertEqual(result.result_set[0][0], 1000000)
+
+        # Change resultset_size from default
+        response = redis_con.execute_command("GRAPH.CONFIG SET RESULTSET_SIZE 2")
+        self.env.assertEqual(response, "OK")
+
+        # Validate modified resultset_size
+        result = redis_graph.query("UNWIND range(1, 10) AS v RETURN v")
+        self.env.assertEqual(len(result.result_set), 2)
+
+        # Revert resultset_size to unlimited with a negative argument
+        response = redis_con.execute_command("GRAPH.CONFIG SET RESULTSET_SIZE -100")
+        self.env.assertEqual(response, "OK")
+
+        # Make sure resultset_size has been updated to unlimited.
+        response = redis_con.execute_command("GRAPH.CONFIG GET RESULTSET_SIZE")
+        expected_response = ["RESULTSET_SIZE", -1]
+        self.env.assertEqual(response, expected_response)
+
+        response = redis_con.execute_command("GRAPH.CONFIG", "GET", "NODE_CREATION_BUFFER")
+        expected_response = ["NODE_CREATION_BUFFER", 16384]
+        self.env.assertEqual(response, expected_response)
+
+    def test09_set_invalid_values(self):
+        # The run-time configurations supported by RedisGraph are:
+        # MAX_QUEUED_QUERIES
+        # TIMEOUT
+        # QUERY_MEM_CAPACITY
+        # DELTA_MAX_PENDING_CHANGES
+        # RESULTSET_SIZE
+
+        # Validate that attempting to set these configurations to
+        # invalid values fails
+        try:
+            # MAX_QUEUED_QUERIES must be a positive value
+            redis_con.execute_command("GRAPH.CONFIG SET MAX_QUEUED_QUERIES 0")
+            assert(False)
+        except redis.exceptions.ResponseError as e:
+            assert("Failed to set config value MAX_QUEUED_QUERIES to 0" in str(e))
+            pass
+
+        # TIMEOUT, QUERY_MEM_CAPACITY, and DELTA_MAX_PENDING_CHANGES must be
+        # non-negative values, 0 resets to default
+        for config in ["TIMEOUT", "QUERY_MEM_CAPACITY", "DELTA_MAX_PENDING_CHANGES"]:
+            try:
+                redis_con.execute_command("GRAPH.CONFIG SET %s -1" % config)
+                assert(False)
+            except redis.exceptions.ResponseError as e:
+                assert("Failed to set config value %s to -1" % config in str(e))
+                pass
+
+        # No configuration can be set to a string
+        for config in ["MAX_QUEUED_QUERIES", "TIMEOUT", "QUERY_MEM_CAPACITY",
+                       "DELTA_MAX_PENDING_CHANGES", "RESULTSET_SIZE"]:
+            try:
+                redis_con.execute_command("GRAPH.CONFIG SET %s invalid" % config)
+                assert(False)
+            except redis.exceptions.ResponseError as e:
+                assert(("Failed to set config value %s to invalid" % config) in str(e))
```

Comment:
I would skip this whole test, the error messages are different and `DELTA_MAX_PENDING_QUERIES` doesn't exist on this version.